samples/snippets/quickstart.py

```diff
@@ -15,87 +15,55 @@
+Before you start the application, please review the comments starting with
+"TODO(developer)" and update the code to use correct values.
 
-2. Provide service account credentials using one of the following options:
-- set the GOOGLE_APPLICATION_CREDENTIALS environment variable, the API
-client will use the value of this variable to find the service account key
-JSON file.
-
-https://cloud.google.com/docs/authentication/production#setting_the_environment_variable
-
-OR
-- manually pass the path to the service account key JSON file to the API client
-by specifying the keyFilename parameter in the constructor:
-https://cloud.google.com/docs/authentication/production#passing_the_path_to_the_service_account_key_in_code
-
-To install the latest published package dependency, execute the following:
-  pip install google-analytics-data
+Usage:
+  pip3 install --upgrade google-analytics-data
+  python3 quickstart.py
 """
-import argparse
+# [START google_analytics_data_quickstart]
+from google.analytics.data_v1beta import BetaAnalyticsDataClient
+from google.analytics.data_v1beta.types import DateRange
+from google.analytics.data_v1beta.types import Dimension
+from google.analytics.data_v1beta.types import Metric
+from google.analytics.data_v1beta.types import RunReportRequest
 
-# [START ga_data_run_report]
-from google.analytics.data_v1alpha import AlphaAnalyticsDataClient
-from google.analytics.data_v1alpha.types import DateRange
-from google.analytics.data_v1alpha.types import Dimension
-from google.analytics.data_v1alpha.types import Entity
-from google.analytics.data_v1alpha.types import Metric
-from google.analytics.data_v1alpha.types import RunReportRequest
 
+def sample_run_report(property_id='YOUR-GA4-PROPERTY-ID'):
+    """Runs a simple report on a Google Analytics 4 property."""
+    # TODO(developer): Uncomment this variable and replace with your
+    #  Google Analytics 4 property ID before running the sample.
+    #property_id = 'YOUR-GA4-PROPERTY-ID'
 
-def sample_run_report(property_id):
-    """Runs a simple report on a Google Analytics App+Web property."""
-
+    # [START google_analytics_data_initialize]
     # Using a default constructor instructs the client to use the credentials
     # specified in GOOGLE_APPLICATION_CREDENTIALS environment variable.
-    client = AlphaAnalyticsDataClient()
-    request = RunReportRequest(entity=Entity(property_id=property_id),
+    client = BetaAnalyticsDataClient()
+    # [END google_analytics_data_initialize]
+
+    # [START google_analytics_data_run_report]
+    request = RunReportRequest(property='properties/' + str(property_id),
                                dimensions=[Dimension(name='city')],
                                metrics=[Metric(name='activeUsers')],
                                date_ranges=[DateRange(start_date='2020-03-31',
                                                       end_date='today')])
```

Comment:
Please make sure the samples pass the lint check. You can run the blacken session to format.

Reply:
Done!